lib/shared/puppet_self_service.rb

```diff
@@ -46,4 +46,72 @@ def self.service_running_enabled(name, service = nil)
 
     service_running(name, service) and service_enabled(name, service)
   end
+
+  # Return active postgres resource in Debian
+  # @return [String] pe-postgres fully qualified service is debian
+  def self.pe_postgres_version
```

Comment:
The naming of this method is odd. The return is not the version, but rather the service name.

Maybe `self.pe_postgres_service_name` or something similar

Reply:
changed